packages/expo-constants/scripts/get-app-config-ios.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+DEST="$CONFIGURATION_BUILD_DIR/$UNLOCALIZED_RESOURCES_FOLDER_PATH"
+NODE_BINARY=${NODE_BINARY:-node}
+
+# Related to: https://github.com/facebook/react-native/blob/c9f869f9c7c8b035a669980382af4bbd4afecb89/scripts/react-native-xcode.sh#L59-L69
+PROJECT_ROOT=${PROJECT_ROOT:-$PWD}
+cd "$PROJECT_ROOT" || exit
+
+if ! [ -x "$(command -v $NODE_BINARY)" ]; then
+  echo 'Error: cannot find the node binary. Try setting the NODE_BINARY variable in the ' \
+  '"Bundle React Native code and images" Build Phase to the absolute path to your node binary. ' \
```

Comment:
i think this should probably use the name of whatever we recommend this build phase be called

Reply:
For expo-updates we just use the existing build phase and append a call to this script to the end. We could probably do the same with this one, unless you think it makes sense to create a separate build phase.

Reply:
ah ok, makes sense